mis_builder_account_move_budget/view/mis_builder.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="mis_report_instance_period_view_form">
+        <field name="name">mis.report.instance.period.view.form</field>
+        <field name="model">mis.report.instance.period</field>
+        <field name="inherit_id"
+               ref="mis_builder.mis_report_instance_period_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='source_data']" position="inside">
+                <field name="source_aml_model_name" invisible="1"/>
+                <field name="account_move_budget_id"
+                       attrs="{'invisible':[('source_aml_model_name', '!=', 'account.move.budget.line')]}"/>
```

Comment:
Could this work with something like `[('source_aml_model_id', '=', ref('model_account_move_budget_line'))]` or something similar? So `source_aml_model_name` field would not be necessary.

Reply:
I couldn't find a way, it might be, but it doesn't seems to be a simple one, it is not possible to use ref there, so I will keep source_aml_model_name for the moment.

Ideas welcomed

Reply:
Yeah, it's probably not possible indeed. Let's add this field in mis_builder then. https://github.com/OCA/mis-builder/pull/214